### ISSUE TITLE
text in articles column is MIA

### ISSUE THREAD

**Issue by IKSGroth** [jangernert/FeedReader#190]
I don't see any text in the right column any more. Images are still there and all the links work (the headline is invisible, the other links appear as green text-decoration lines). The other two columns aren't affected.

The problem occured on two Fedora installations after a system update (normal package update). I don't know if this is even a bug in FeedReader or if the problem is caused solely by the Fedora package or the rest of my Fedora installations. As far as I can tell, the FeedReader package wasn't updated on one of the installations. 

OS: Fedora 24
Package: feedreader-1.6.1-1.fc24.x86_64
Server: Nextcloud 10 / news 9.0.4

I might file a bug in the RedHat bugzilla later, it's just that I don't have any clue what the problem is.

**Comment by jangernert** [comment]
Was webkitgtk4 updated?
FeedReader was not updated as far as I am aware of.

**Comment by jangernert** [comment]
updated my fedora box and have the same issue now =/
webkitgtk4 package was updated

**Comment by jangernert** [comment]
WebKit.WebView.load_html() base_uri parameter seems to be broken:

This works but obviously doesn't show any images:
```
		m_currentView.load_html(
			Utils.buildArticle(
					Article.getHTML(),
					Article.getTitle(),
					Article.getURL(),
					Article.getAuthor(),
					Article.getDateNice(),
					Article.getFeedID()
				)
			, null);
```

This triggers the bug:
```
		m_currentView.load_html(
			Utils.buildArticle(
					Article.getHTML(),
					Article.getTitle(),
					Article.getURL(),
					Article.getAuthor(),
					Article.getDateNice(),
					Article.getFeedID()
				)
			, "file://" + GLib.Environment.get_home_dir() + "/.local/share/feedreader/data/images/");
```

**Comment by jangernert** [comment]
Turns out it was due to some fonts in the CSS-code.
I released a hotfix [1.6.2](https://github.com/jangernert/FeedReader/releases/tag/v1.6.2)
